android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java

```diff
@@ -444,9 +444,8 @@ private boolean hasGradient(KrollDict d)
 
 	private boolean hasBorder(KrollDict d)
 	{
-		return d.containsKeyAndNotNull(TiC.PROPERTY_BORDER_COLOR)
-			|| d.containsKeyAndNotNull(TiC.PROPERTY_BORDER_RADIUS)
-			|| d.containsKeyAndNotNull(TiC.PROPERTY_BORDER_WIDTH);
+		return d.containsKeyAndNotNull(TiC.PROPERTY_BORDER_WIDTH)
+			&& d.getDouble(TiC.PROPERTY_BORDER_WIDTH) > 0f;
```

Comment:
I believe we still need to do an -or- check to see if radius is `> 0f` because it'll "clip" the view with rounded corners, even if the border width is zero, right?